app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -485,13 +485,52 @@ public void onResume() {
         registerNetworkReceiver();
         if (isResumed() && ((MainActivity)getActivity()).activeFragment == ActiveFragment.NEARBY) {
             if(!isPermissionDenied && !applicationKvStore.getBoolean("doNotAskForLocationPermission", false)){
-                startTheMap();
+                if (!locationManager.isGPSProviderEnabled()) {
+                    startMapWithoutGPS();
+                } else {
+                    startTheMap();
+                }
             }else{
                 startMapWithoutPermission();
             }
         }
     }
 
+    /**
+     * Starts the map without GPS
+     * By default it points to 51.50550,-0.07520 this coordinates
+     */
+    private void startMapWithoutGPS() {
```

Comment:
These two methods look very similar, To avoid repetition, I think it is better to combine them into a single method and pass the appropriate arguments.

![image](https://user-images.githubusercontent.com/30932899/152540990-5855aaf8-20fb-4356-b2b6-b508085ed15d.png)